packages/browser/src/sdk.ts

```diff
@@ -97,9 +105,17 @@ export function init(options: BrowserOptions = {}): void {
   if (options.stackParser === undefined) {
     options.stackParser = defaultStackParsers;
   }
-
   const { transport, newTransport } = setupBrowserTransport(options);
-  initAndBind(BrowserClient, options, transport, newTransport);
+
+  const clientOptions: BrowserClientOptions = {
+    ...options,
+    ...defaultCoreOptions,
+    stackParser: stackParserFromOptions(options),
+    integrations: getIntegrationsToSetup(options),
```

Comment:
this + the client options changes means the integrations are now treeshakable :)